dev/com.ibm.ws.classloading/resources/com/ibm/ws/classloading/internal/resources/ClassLoadingServiceMessages.nlsprops

```diff
@@ -139,3 +139,19 @@ slf.no.acf.useraction=See the problem determination information on the WebSphere
 slf.failed.delete=CWWKL0072W: The [{0}] cache file cannot be deleted. This result can lead to the corruption of nested archives.
 slf.failed.delete.explanation=The system failed to delete the cache file. Because the file still exists in the system, this might cause corruption in nested archives.
 slf.failed.delete.useraction=Verify that the cache file is not in a locked state, which can cause the file deletion to fail.
+
+cls.classloader.config.duplicate=CWWKL0080E: API type [{0}] was already used in this list [{1}], remove the duplicate API type.
```

Comment:
Global comment: Remember to use the variable before the object type, and start a new sentence when you have two independent clauses. For example, write:

The [{0}] API type was already used in the [{1}] list. Remove the duplicate API type.

Reply:
Completed changes